binaries/apiserver/load_test/http_api.go

```diff
@@ -0,0 +1,100 @@
+package load_test
+/*
+start the load testers as an http server exposing the endpoints:
+get apiserver_test - returns the status (job running, waiting to start, etc.)
+get apiserver_test?action=<upload/download>&... - starts a load test
+get apiserver_test/kill - kills the current running test
+ */
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func (lt *ApiserverLoadTester) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case "GET":
+		if strings.Contains(r.RequestURI, "/kill") {
+			lt.KillEndpoint(w, r)
+		} else {
+			lt.GetEndpoint(w, r)
+		}
+	default:
+		resp := fmt.Sprintf("Method %s not implemented", r.Method)
+		json.NewEncoder(w).Encode(resp)
+	}
+}
+
+func (lt *ApiserverLoadTester) GetEndpoint(w http.ResponseWriter, r *http.Request) {
```

Comment:
Somewhat nit for a load tester - seems confusing having GET used to start tests - can we make this PUT or POST? (same with KillEndpoint (DELETE?))

Reply:
agreed, I had it as POST then realized that I can't pass params in on url (using curl).   I moved to to get just to simplify curling.  If I make it POST we'll need to use -H "Content-Type: application/json"  and -d '{<param def json>}' in curl statements.

if we leave it as GET I can add the above as a comment.

Reply:
Ok, get + comment sounds fine

Reply:
added comment